Dockerfile

```diff
@@ -14,6 +14,8 @@ ADD node_modules node_modules/
 RUN npm run-script collect-static
 ADD static static/
 
+RUN mkdir -m 777 sessions
```

Comment:
Just started to have a look at this so I don't know very well who will access `sessions` but do we need this level of permissions?

Who will read or write from this directory? If it's just the current user [`600 / (only owner can read/write)`](http://www.filepermissions.com/file-permission/600) may be more than plenty.

Reply:
I meant to remove this, actually - we're not storing sessions in the filesystem